index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en-us">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Zema's Portfolio</title>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <div class="status-bars">
+
+    </div>
+    <div class="app-bar">
+      <div class="toolbar">
+        <div class="normal-button">
+          <img class="burger" src="assets/burger.svg" alt="Menu icon.">
+        </div>
+        <div class="logo">
+          <span>My Logo</span>
+        </div>
```

Comment:
- [ ] Kindly wrap the logo in an anchor `<a>` tag, as it will serve as a direct one-click link to the homepage

Reply:
@phelian23 Done! Can you check, please?